test/e2e/kubernetes/deployment/deployment.go

```diff
@@ -131,6 +146,29 @@ func CreateWindowsDeploy(image, name, namespace string, port int, hostport int)
 	return d, nil
 }
 
+// CreateWindowsDeployIfNotExist first checks if a deployment already exists, and return it if so
```

Comment:
Nitpick: Go [doc comments](https://golang.org/doc/effective_go.html#commentary) should end with periods.